src/core/client/logger.js

```diff
@@ -1,58 +0,0 @@
-/* eslint-disable no-console */
-/* global window */
-
-import config from 'config';
-
-/*
- * This is the client version of the logger.
- * This module loader is loaded when importing 'core/logger' in the client.
- */
-
-export function bindConsoleMethod(
-  consoleMethodName,
-  {
-    _consoleObj = window.console,
-    _function = Function,
-    _noop = () => {},
-    _config = config,
-  } = {},
-) {
-  if (typeof _consoleObj[consoleMethodName] === 'undefined') {
-    throw new Error(`console method "${consoleMethodName}" does not exist`);
-  }
-
-  if (_config.get('enableClientConsole') === false) {
-    return _noop;
-  }
-
-  let consoleFunc;
-  const appName = _config.get('appName');
-  const consoleMethod = _consoleObj[consoleMethodName];
-
-  const app = `[${appName}]`;
-
-  if (_function.prototype.bind) {
-    consoleFunc = _function.prototype.bind.call(
-      consoleMethod,
-      _consoleObj,
-      app,
-    );
-  } else {
-    // Fallback for IE < 10;
```

Comment:
Do you know what browsers pino supports? I can't find anything definitive on their readme.

Reply:
heh, looks like IE 10 might not work: https://github.com/pinojs/pino/issues/311.

Reply:
I think it works for IE10 and more: https://github.com/pinojs/pino/pull/323, which looks ok to me (according to Analytics, we had 0.36% users on Internet Explorer from January 1st, 2018).